src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx

```diff
@@ -84,6 +86,16 @@ export const View = (props: Props) => {
   const l10n = useL10n();
   const recordTelemetry = useTelemetry(props.experimentationId);
   const countryCode = useContext(CountryCodeContext);
+  const pathname = usePathname();
+
+  const [activeTab, setActiveTab] = useState<TabType>(props.activeTab);
+
+  useEffect(() => {
+    const nextPathname = `/user/dashboard/${activeTab}`;
+    if (pathname !== nextPathname) {
```

Comment:
```suggestion
    if (pathname !== nextPathname) {
      // Directly interacting with the history API is recommended by Next.js to avoid re-rendering on the server. See
      // https://github.com/vercel/next.js/discussions/48110#discussioncomment-7563979
```

Reply:
Thanks, added the comment in [ae2d1a5](https://github.com/mozilla/blurts-server/pull/4477/commits/ae2d1a5b66d2472dc8069aa79711d567f7fd5b0d).